openvpn/fetch-external-configs.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+
+VPN_CONFIG_SOURCE_TYPE="${VPN_CONFIG_SOURCE_TYPE:-github_zip}"
+
+# Set default GitHub config repo
+GITHUB_CONFIG_SOURCE_REPO="${GITHUB_CONFIG_SOURCE_REPO:-haugene/vpn-configs}"
+GITHUB_CONFIG_SOURCE_REVISION="${GITHUB_CONFIG_SOURCE_REVISION:-main}"
+
+if [[ "${VPN_CONFIG_SOURCE_TYPE}" == "github_zip" ]]; then
+
+  # Concatenate URL for config bundle from the given GitHub repo
+  GITHUB_CONFIG_BUNDLE_URL="https://github.com/${GITHUB_CONFIG_SOURCE_REPO}/archive/${GITHUB_CONFIG_SOURCE_REVISION}.zip"
+  
+  # Create a temporary file and download bundle to it
+  config_repo_temp_zip_file=$(mktemp)
+  echo "Downloading configs from ${GITHUB_CONFIG_BUNDLE_URL} into ${config_repo_temp_zip_file}"
+  curl -sSL -o "${config_repo_temp_zip_file}" "${GITHUB_CONFIG_BUNDLE_URL}"
+
+  # Create a temporary folder and extract configs there
+  config_repo_temp_dir=$(mktemp -d)
+  echo "Extracting configs to ${config_repo_temp_dir}"
+  unzip -q "${config_repo_temp_zip_file}" -d "${config_repo_temp_dir}"
```

Comment:
missing check of unzip success

Reply:
I set
```
set -o nounset
set -o errexit
set -o pipefail
```
in the script as proposed. it now fails if the unzip command fails.